code/defines/procs/statistics.dm

```diff
@@ -1,15 +1,5 @@
-/proc/sql_report_round_start()
-	// TODO
-	if(!config.sql_enabled)
-		return
-
-/proc/sql_report_round_end()
-	// TODO
-	if(!config.sql_enabled)
-		return
-
 /proc/sql_report_death(mob/living/carbon/human/H)
-	if(!config.sql_enabled)
+	if(!SSdbcore.IsConnected())
```

Comment:
Same as above. It does a rust call each time

Reply:
Same as above.